editors/mediawiki.el/Portfile

```diff
@@ -1,34 +1,36 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
 PortSystem 1.0
+PortGroup github 1.0
```

Comment:
please align these two lines the same as the rest of the Portfile (to conform to the modeline).

Reply:
ok, done in ba25bf1